Havit.Blazor.Documentation/Pages/Components/FormInputs/FormInputs_Demo_CustomValidation.razor

```diff
@@ -0,0 +1,65 @@
+﻿<EditForm Model="@_testField" OnValidSubmit="@(() => model.Text = "Validation passed... :-)")">
+	<DataAnnotationsValidator />
+
+    <HxInputText Label="HxInputText" @bind-Value="@_testField.TestText" ValidationEditContext="new(_testField.TestModel)" ValidationFor="() => _testField.TestModel.Text"
+        Placeholder="Enter text here." Hint="Hint: Enter text, the value is shared with all text inputs." />
```

Comment:
This seems a bit unclear:

* The current `<EditForm />` uses `_testField` as the `Model`, while the inputs bind to `@model` properties. Why not use `<EditForm Model="@model">` instead?
* The new `ValidationXy` will cause the form to display a validation message that the user cannot resolve. Is there no way to set `_testField.TestModel.Text`? Additionally, the `HxInputText` will still trigger validation on the bound property, even if the `HxValidationMessage` is redirected elsewhere.
* Have you considered using nested `<EditForm />` components for the different models used by `HxInputText` in this context?
* Have you thought about using `<HxValidationMessage />` directly, combined with `<HxInputText ValidationMessageMode="ValidationMessageMode.None" />` for inputs where you don't want to expose their validation messages?

@mmonteagudo I might not fully understand the intent here. Could you provide a real-world scenario to clarify?

Reply:
I want able to provide it for the examples in the code, but in my case I'm using it for Validating using Fluent Validation when validating values that are not parameters, for example in my case:

One if my properties in my project is ApplicationData which is a Dictionary, and I need to validate when one for the items in the dictionary has an specific value as a condition to validate all others, as default validation only works with properties, that's not possible, so in this in case I declared in Fluent Validation a "custom property to validate" and I'm matching that property (in this case equals to the dictionary key)

So, basically I'm sending to Fluent a custom property name to validate.

Hope the explanation helps, here are some code examples where I'm using the implementation

![image](https://github.com/user-attachments/assets/63032a02-695f-4f1d-9099-739b6ce4139a)

![image](https://github.com/user-attachments/assets/fde8c544-09e5-48f7-bfd4-aef641173667)

Reply:
Custom validation feature removed from the PR.